lib/new_relic/agent/log_event_aggregator.rb

```diff
@@ -174,7 +189,9 @@ def after_harvest metadata
       # To avoid paying the cost of metric recording on every line, we hold
       # these until harvest before recording them
       def record_customer_metrics
+        return unless enabled?
         return unless NewRelic::Agent.config[:'application_logging.metrics.enabled']
```

Comment:
Metrics can be disabled separately? So someone could have logs enabled and sending them up but have the metrics for it disabled? That seems pretty different from most of what the agent usually does, so I'm just curious

Reply:
Yes. It seems odd to me, but that is one of the core logging features. Supportability metrics will still be sent up. The metrics that won't be sent are `Logging/lines` and `Logging/lines/{severity}`. I believe this was added because some people may want to fine-tune their performance and adding incrementing metrics like this have more of a performance impact in other language agents.

Reply:
(discussed via mobbing)